packages/go_router/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: go_router
+description: A declarative router for Flutter based on Navigation 2 supporting
+  deep linking, data-driven routes and more
+version: 3.0.1
+repository: https://github.com/flutter/packages/tree/master/packages/go_router
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+go_router%22
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.17.0"
+
+dependencies:
+  collection: ^1.15.0
+  flutter:
+    sdk: flutter
+  flutter_web_plugins:
+    sdk: flutter
+  logging: ^1.0.0
+  path_to_regexp: ^0.4.0
+
+dev_dependencies:
+  all_lint_rules_community: ^0.0.4
```

Comment:
We need to remove this as well. It's not only not a 1P package, but it could add new warnings and break our build without warning since it's not pinned.

We can inline all of the options as the short-term solution until we migrate to the correct analysis options.